src/styleguide/_patterns/02-molecules/00-navigation/00-nav-menu.mustache

```diff
@@ -1,5 +1,5 @@
 <button type="button" class="grav-c-toggle-menu" aria-pressed="{{control.pressed}}">
-  {{control.label}}
+  <span class="grav-c-toggle-menu__icon"></span>
```

Comment:
would it be useful to move this tag into a separate atom? 
in which case you have two solutions: 
1. create a normal pattern and show it in the list of atoms/icons in patternlab
2. create a hidden pattern (prefixing it with `_` underscore)